### PR TITLE
Bump to Android API 30

### PR DESCRIPTION
**Pull request by Cervator**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ RUN mkdir -p /opt/android-sdk/cmdline-tools \
 && mv /opt/android-sdk/cmdline-tools/cmdline-tools /opt/android-sdk/cmdline-tools/latest \
 && mkdir /opt/android-sdk/licenses \
 && printf "\n24333f8a63b6825ea9c5514f83c2829b004d1fee" > /opt/android-sdk/licenses/android-sdk-license \
-&& /opt/android-sdk/cmdline-tools/latest/bin/sdkmanager "build-tools;29.0.2" "platforms;android-28" \
+&& /opt/android-sdk/cmdline-tools/latest/bin/sdkmanager "build-tools;30.0.2" "platforms;android-30" \
```
Comment (BenjaminAmos) [inline Dockerfile]: The latest patch release for the API 30 build tools is `30.0.3`, so maybe use that instead?
```suggestion
&& /opt/android-sdk/cmdline-tools/latest/bin/sdkmanager "build-tools;30.0.3" "platforms;android-30" \
```

**Review by BenjaminAmos — APPROVED** [review]
These changes seem sensible to me. Thanks for correcting the `platforms;android` version.

File: Dockerfile (hunk 1 above)
Comment (Cervator) [reply Dockerfile]: Sure thing, I pretty much just googled an example and copy pasted 😁

**Comment by BenjaminAmos** [comment]
You might want to update the instructions in the `DestSolAndroid` [README](https://github.com/MovingBlocks/DestSolAndroid#using-the-android-facade) to reference the new SDK version too.

**Comment by Cervator** [comment]
Done!

**Pull request merged**